sentry/src/main/java/io/sentry/MainEventProcessor.java

```diff
@@ -72,6 +75,21 @@
     return event;
   }
 
+  private void setDebugMeta(final @NotNull SentryEvent event) {
+    if (event.getDebugMeta() == null && !options.getProguardUuids().isEmpty()) {
```

Comment:
let's remove this check `event.getDebugMeta()` and instantiate if its null when `getProguardUuids` isn't empty, the reason is, an Android event may have native images, so it's not gonna be null but still has to be appended

Reply:
👍